src/test/system_tests/ceph_s3_tests/test_ceph_s3_deploy.sh

```diff
@@ -24,6 +24,11 @@ if [ ! -d $DIRECTORY ]; then
     cd ${DIRECTORY}
     git checkout ${CEPH_TESTS_VERSION}
     echo "Finished Downloading Ceph S3 Tests"
+
+    echo "Manually fixing s3select tests"
+    sed -i '14 i from . import get_new_bucket_name' ./s3tests_boto3/functional/test_s3select.py
+    sed -i 's/bucket_name = \"test\"/bucket_name = get_new_bucket_name()/g' ./s3tests_boto3/functional/test_s3select.py
```

Comment:
Would you please add a comment on what the fix is (regex are not easy to understand when reading the code) and why we add them?